pebble/src/main/java/io/pebbletemplates/pebble/extension/core/ConfigurableExtensionCustomizer.java

```diff
@@ -0,0 +1,147 @@
+package io.pebbletemplates.pebble.extension.core;
+
+import io.pebbletemplates.pebble.extension.Extension;
+import io.pebbletemplates.pebble.extension.ExtensionCustomizer;
+import io.pebbletemplates.pebble.extension.Filter;
+import io.pebbletemplates.pebble.extension.Test;
+import io.pebbletemplates.pebble.operator.BinaryOperator;
+import io.pebbletemplates.pebble.operator.UnaryOperator;
+import io.pebbletemplates.pebble.tokenParser.TokenParser;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+/**
+ * @author freshchen
+ * @since 2024/3/2
+ */
+public class ConfigurableExtensionCustomizer {
+
+  private Collection<String> disallowedFilterKeys;
+
+  private Collection<String> disallowedTokenParserTags;
+
+  private Collection<String> disallowedFunctionKeys;
+
+  private Collection<String> disallowedBinaryOperatorSymbols;
+
+  private Collection<String> disallowedUnaryOperatorSymbols;
+
+  private Collection<String> disallowedTestKeys;
+
+
+  public static ConfigurableExtensionCustomizer builder() {
+    return new ConfigurableExtensionCustomizer();
+  }
+
+  public ConfigurableExtensionCustomizer setDisallowedFunctionKeys(Collection<String> disallowedFunctionKeys) {
```

Comment:
remove set prefix from all method name of the builder

Reply:
good suggestion